src/overlays/actors/ovl_En_Karebaba/z_en_karebaba.c

```diff
@@ -1,75 +1,523 @@
+    CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_en_karebaba.c", 1034);
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Karebaba/func_80A95BFC.s")
+#ifdef NON_MATCHING
+void EnKarebaba_Draw(Actor* thisx, GlobalContext* globalCtx) {
+    f32 scale;
+    Gfx** current;
+    s32 tmp, i;
+    EnKarebaba* this = THIS;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Karebaba/func_80A95D90.s")
+    OPEN_DISPS(globalCtx->state.gfxCtx, "../z_en_karebaba.c", 1056);
+    func_80093D18(globalCtx->state.gfxCtx);
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Karebaba/func_80A96048.s")
+    if (this->actionFunc == EnKarebaba_DeadItemDrop && (this->actor.params >= 41 || (this->actor.params & 1) != 0)) {
+        Matrix_Translate(0.0f, 0.0f, 200.0f, 1);
+        gSPMatrix(oGfxCtx->polyOpa.p++, Matrix_NewMtx(globalCtx->state.gfxCtx, "../z_en_karebaba.c", 1066), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+        gSPDisplayList(oGfxCtx->polyOpa.p++, UNK_DLIST);
+    } else if (this->actionFunc != EnKarebaba_Wait) {
+        func_80026230(globalCtx, &BLACK_COLOR, 1, 2);
+        SkelAnime_Draw(globalCtx, this->skelAnime.skeleton, this->skelAnime.limbDrawTbl, 0, 0, 0);
+        Matrix_Translate(this->actor.posRot.pos.x, this->actor.posRot.pos.y, this->actor.posRot.pos.z, 0);
+        scale = (this->actionFunc == EnKarebaba_Regrow || this->actionFunc == EnKarebaba_Grow) ? (this->actor.params * 0.0005f) : 0.01f;
+        Matrix_Scale(scale, scale, scale, 1);
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Karebaba/func_80A960B0.s")
+    Matrix_RotateRPY(this->actor.shape.rot.x, this->actor.shape.rot.y, 0, 1);
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Karebaba/func_80A96184.s")
+    tmp = this->actionFunc == EnKarebaba_Dying ? 2 : 3;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Karebaba/func_80A961DC.s")
+    for (i = 0; i < tmp; i++) {
+        current = DisplayLists[i];
+        Matrix_Translate(0, 0, -2000.0f, 1);
+        gSPMatrix(oGfxCtx->polyOpa.p++, Matrix_NewMtx(globalCtx->state.gfxCtx, "../z_en_karebaba.c", 1116), G_MTX_LOAD | G_MTX_NOPUSH | G_MTX_MODELVIEW);
+        gSPDisplayList(oGfxCtx->polyOpa.p++, *current);
+        if (current == &DisplayLists[0] && this->actionFunc == EnKarebaba_Dying) {
+            Matrix_MultVec3f(&VEC_ZERO2, &this->actor.posRot2.pos);
+        }
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Karebaba/EnKarebaba_Update.s")
+    func_80026608(globalCtx);
+    func_80026230(globalCtx, &BLACK_COLOR, 1, 2);
+    Matrix_Translate(this->actor.initPosRot.pos.x, this->actor.initPosRot.pos.y, this->actor.initPosRot.pos.z, 0);
+    if (this->actionFunc != EnKarebaba_Grow) {
+        scale = 0.01f;
+    }
+    Matrix_Scale(scale, scale, scale, 1);
+    Matrix_RotateY(this->actor.initPosRot.rot.y * 0.0000958738f, 1);
```

Comment:
`0.0000958738f` should match as `(M_PI / 0x8000)` (may need `(M_PI / (f32)0x8000)` to match)